### PR TITLE
port echo rdepends:py36-cryptography Remove Python 36 dependencies

### PR DESCRIPTION
**Pull request by essandess**
* Fixes: https://github.com/macports/macports-ports/pull/15044#issuecomment-1145875399

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS x.y
Xcode x.y / Command Line Tools x.y.z

###### Verification 
Have you

- [ ] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [ ] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [ ] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [ ] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?

### CODE REVIEW

**Review by mascguy — APPROVED** [review]
LGTM.

@stromnov @catap @kurthindenburg @petrrr @rpfisher @MarcusCalhoun-Lopez ?

File: python/py-lscsoft-glue/Portfile [hunk 1]
```diff
@@ -25,7 +25,7 @@ checksums            rmd160  98055a86e6c2f5eae1749338e4d895204eef121e \
                      sha256  9bdfaebe4c921d83d1e3d1ca24379a644665e9d7530e7070665f387767c66923 \
                      size    1618606
 
-python.versions      27 36 37 38
```
Comment (reneeotten) [inline python/py-lscsoft-glue/Portfile:28]: this has dependents:
```
% port echo depends:py36-lscsoft-glue
py36-lal
py36-lalburst
py36-lalinspiral
```

File: python/py-oauthlib/Portfile [hunk 2]
```diff
@@ -25,7 +25,7 @@ checksums           rmd160  1ee81d217311e971298be031730cc72a5dc4bafd \
                     sha256  8f0215fcc533dd8dd1bee6f4c412d4f0cd7297307d43ac61666389e3bc3198a3 \
                     size    161395
 
-python.versions     27 36 37 38 39 310
```
Comment (reneeotten) [inline python/py-oauthlib/Portfile:28]: this has dependents:
```
% port echo depends:py36-oauthlib
py36-requests-oauthlib
```

File: python/py-openssl/Portfile [hunk 3]
```diff
@@ -25,7 +25,7 @@ checksums           rmd160  5806276d4716458a77c32f91a0c7f8ee30fe1dfb \
                     sha256  5d1d93fb3d55be740e444b991bccc5db26b1cfd3666cbbd8b6d01dc7a6cc5430 \
                     size    172965
 
-python.versions     27 36 37 38 39 310
```
Comment (reneeotten) [inline python/py-openssl/Portfile:28]: this has dependents:
```
% port echo depends:py36-openssl
py36-lscsoft-glue
py36-service_identity
py36-libcloud
```

**Pull request merged**